pages/projects_new.tsx

```diff
@@ -1,15 +1,128 @@
-import { Typography, useTheme } from '@mui/material'
+import { Typography, Stack, useTheme } from '@mui/material'
 import { withBasicLayout } from 'components/layouts'
+import CardGridContainer from 'components/cards/CardGridContainer'
+import CardProject from 'components/cards/CardProject'
+import useMediaQuery from '@mui/material/useMediaQuery'
+
+import Placeholder from '../assets/placeholder-img.png'
 
 const ProjectsPage = () => {
   const theme = useTheme()
 
+  const extraSmallScreen = useMediaQuery(theme.breakpoints.only('xs'))
+
+  const projectData = [
```

Comment:
I like how you handled working with project information for the time being!
@santina and @mistyb01 we should make a todo to discuss content management and how we want to go about empowering cadre members to add content to the site without coding. For example, do we want to use Sanity since it is already set up, or use another excellent tool that someone has positive experience with?